storagenode/console/service.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package console
+
+import (
+	"context"
+	"time"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/pkg/kademlia"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/bandwidth"
+	"storj.io/storj/storagenode/pieces"
+)
+
+// Service is handling storage node operator related logic
+type Service struct {
+	log *zap.Logger
+
+	bandwidth bandwidth.DB
+	pieceInfo pieces.DB
+	kademlia  *kademlia.Kademlia
+
+	allocatedBandwidth memory.Size
+	allocatedDiskSpace memory.Size
```

Comment:
*monitor.Service has the necessary information already, so rather ask that service.

Reply:
@egonelbre but, as i understand, the main purpose of monitoring service - is to update kademlia in loop. It is separate service with defined responsibility.

In our case we created new service, main responsibility of which - is to return data to our web ui

Won't we break SRP if we will use monitor for this purpose? 

Moreover, we have some specific requirements in SNO operator. And we will have more than one reason to modify monitoring service in future each time when we will receive new specifications for SNO dashboard

Reply:
Monitoring service is responsible for monitoring the storage node disk usage, bandwidth etc. and updating kademlia.

I didn't mean that `console.Service` should be removed, but rather that it can rely `monitor.Service` and other services for the information about the disk usage / bandwidth information. That way we don't duplicate the logic and accidentally end up displaying different information for dashboard and the console.

Reply:
But I get your point, the responsibilities at the moment are a bit fuzzy.